### ISSUE TITLE
Certificate SANS Fields not displayed

### ISSUE THREAD

**Issue by madeinoz67** [WesBunton/MyCertificates#12]
I noticed that MyCertificates does not display SANS fields.  this would be helpful especially when device certificates for EAP-TLS are deployed and need to be reviewed

**Comment by rossoneris** [comment]
Yes - SAN field could really be needed if someone still maintain this app
